corehq/apps/translations/app_translations/upload_module.py

```diff
@@ -283,7 +251,33 @@ def _update_details_based_on_position(self, list_rows, short_details, detail_row
             self._update_detail(row, detail)
 
     def _partial_upload(self, rows, details):
+        expected_fields = [detail.field for detail in details]
+        received_fields = [row['id'] for row in rows]
+        message = _('The following rows are missing for menu {index}: {fields}').format(
+            index=self.module.id + 1,
+            fields=', '.join(set(expected_fields) - set(received_fields))
+        )
+        self.msgs.append((messages.info, message))
```

Comment:
Not quite sure what the user should think or do when they see this message. If it's notifying that the fields not in the upload won't be updated, I think we can leave it off.